backend/prisma/migrations/20240713163709_thought/migration.sql

```diff
@@ -0,0 +1,13 @@
+-- CreateTable
+CREATE TABLE "Thought" (
+    "id" TEXT NOT NULL,
+    "content" TEXT NOT NULL,
+    "publishedDate" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "published" BOOLEAN NOT NULL DEFAULT false,
+    "authorId" TEXT NOT NULL,
```

Comment:
```javascript
 author        User       @relation(fields: [authorId], references: [id], onDelete: Cascade)
 authorId      String
 ```

Reply:
@aadeshkulkarni what should I change here?